perception/sandbox/polygrasp/src/polygrasp/cam_pub_sub.py

```diff
@@ -0,0 +1,126 @@
+import logging
+import json
+from types import SimpleNamespace
+
+import numpy as np
+import open3d as o3d
+
+import a0
+import realsense_wrapper
+from polygrasp import serdes
+
+log = logging.getLogger(__name__)
```

Comment:
How do you decide when to `print` and when to `log.info`?

Reply:
I tend to use the logger when it's library code (easier to find the source when something goes wrong) and print in user code / scripts (by default / for iteration speed). Any feedback on this?